_rules/element-lang-matches-default-off6ek.md

```diff
@@ -0,0 +1,337 @@
+</html>
+```
+
+#### Passed Example 4
+
+This `div` element has a `lang` attribute value of `en` (English), which matches its [default language][]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a (correct) `lang` attribute value of `fr`.
+
+```html
+<html lang="fr">
+	<head>
+		<title>Feu d'artifice du nouvel an</title>
+	</head>
+	<body>
+		<div lang="en">
+			<img src="/test-assets/shared/fireworks.jpg" alt="Fireworks over Paris" />
+			<p lang="fr">
+				Bonne année !
+			</p>
+		</div>
+		<p>
+			Un feu d'artifice sera tiré depuis les jardins du Trocadéro.
+		</p>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `span` element has `lang` attribute value of `fr` (French), which does not matches its [default language][]. The default language is Dutch because all words are Dutch.
+
+```html
+<html lang="en">
+	<head>
+		<title>Dutch idioms</title>
+	</head>
+	<body>
+		<p>
+			The Dutch phrase <span lang="fr">"Hij ging met de kippen op stok"</span> literally translates into "He went to
+			roost with the chickens", but it means that he went to bed early.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+The second `p` element has a `lang` attribute value of `fr` (French), which does not match its [default language][]. The default language is English because all but a few words are English.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="fr">
+			The Dutch phrase "Hij ging met de kippen op stok" literally translates into "He went to roost with the chickens",
+			but it means that he went to bed early.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+The second `p` element has `lang` attribute value of `en` (English), which does not matches its [default language][]. The default language is Dutch because all English words are in `span` elements with a `lang` attribute value of `fr`. Both `span` elements also have an incorrect `lang` attribute in order to make sure that all targets in this example fail the rule.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			<span lang="fr">The Dutch phrase</span> "Hij ging met de kippen op stok"
+			<span lang="fr"
+				>literally translates into "He went to roost with the chickens", but it means that he went to bed early.</span
+			>
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+This `div` element has a `lang` attribute value of `fr` (French), which does not match its [default language][]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `fr`. The first `p` element also has an incorrect `lang` attribute.
```

Comment:
```suggestion
This `div` element has a `lang` attribute value of `fr` (French), which does not match its [default language][]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `nl` which is also an incorrect `lang` attribute.
```